src/satellite/test.js

```diff
@@ -103,7 +102,8 @@ describe('Satellite()', () => {
       service.start(port, async () => {
         const res = await fetch(`${url}/healthcheck`);
         expect(res.ok).toBe(false);
-        service.stop(done);
+        await service.stop();
```

Comment:
The other way to do this would be to pass back the Promise this returns to Jest, and it will wait on it:

```js
test('...', async () => {
  ...
  return service.stop();
});
```